sdk/core/loader/debug.hh

```diff
@@ -132,8 +132,8 @@ namespace
 		/**
 		 * Append a capability.
 		 */
-		template<typename T>
-		__always_inline void append(CHERI::Capability<T> capability)
+		template<typename T, bool IsSealed>
+		__always_inline void append(CHERI::Capability<T, IsSealed> capability)
```

Comment:
I wonder if it's worth a check and warning if the sealed state of the capability differs from the static annotation?

Reply:
That would need to be in the debug library and would need us to use a different type to pass in sealed things, but is probably a good idea.